### DISCUSSION TITLE
Writing test for prebuilt actions

### DISCUSSION THREAD

**Discussion by dimmm931** [filamentphp/filament#11057] [Help]
### Package

Actions

### Package Version

3.2

### How can we help you?

I want to write a test for pre built actiosn, let us say for Export action. I follow the  test guideline https://filamentphp.com/docs/3.x/actions/testing, but it requires specifying the name of action **->callAction('NAME')**, where can I get it?

**Comment by leandrocfe** [comment]
via `getDefaultName()`

```php
ExportAction::getDefaultName()
```

**Reply by phh** [reply]
And using the `assertTableBulkActionExists()` instead of `assertActionExists()`.
